kernel/main.c

```diff
@@ -359,6 +482,8 @@ kmain(u32 multiboot_magic, u32 mbi_addr)
    init_kmalloc();
    init_paging();
 
+   setup_uefi_runtime_services();
+
```

Comment:
No extra line between `setup_uefi_runtime_services();` and `acpi_mod_init_tables();`